themes/vasuki-dark-theme.json

```diff
@@ -1,589 +1,560 @@
-				"foreground": "#b267e6"
-			}
-		}
-	]
-}
\ No newline at end of file
+  "name": "Vasuki Dark Theme",
+  "type": "dark",
+  "colors": {
+    "focusBorder": "#0C0E14",
+    "input.background": "#171a26",
+    "dropdown.background": "#0C0E14",
+    "dropdown.border": "#293045",
+    "list.focusBackground": "#383e4a",
+    "list.activeSelectionBackground": "#141721",
+    "list.activeSelectionForeground": "#d7dae0",
+    "list.inactiveSelectionBackground": "#141721",
+    "list.inactiveSelectionForeground": "#d7dae0",
+    "list.hoverBackground": "#141721",
+    "list.highlightForeground": "#c5c5c5",
+    "button.background": "#528bff",
+    "scrollbarSlider.background": "#4e566680",
+    "scrollbarSlider.hoverBackground": "#5a637580",
+    "scrollbarSlider.activeBackground": "#747d9180",
+    "editor.background": "#0C0E14",
+    "editorWidget.background": "#0C0E14",
+    "editor.selectionBackground": "#3e4451",
+    "editor.findMatchHighlightBackground": "#314365",
+    "editorHoverWidget.background": "#0C0E14",
+    "editorHoverWidget.border": "#293045",
+    "diffEditor.insertedTextBackground": "#00809b33",
+    "editor.lineHighlightBackground": "#0C0E14",
+    "editorCursor.foreground": "#f8f8f0",
+    "editorWhitespace.foreground": "#484a50",
+    "editorIndentGuide.background": "#3b4048",
+    "editorLineNumber.foreground": "#495162",
+    "editorRuler.foreground": "#484848",
+    "editorError.foreground": "#704040",
+    "editorWarning.foreground": "#407040",
+    "editorInfo.foreground": "#707040",
+    "editorSuggestWidget.background": "#0C0E14",
+    "editorSuggestWidget.border": "#293045",
+    "editorSuggestWidget.selectedBackground": "#2c313a",
+    "peekView.border": "#293045",
+    "peekViewEditor.background": "#202020",
+    "tab.activeBackground": "#141721",
+    "tab.inactiveBackground": "#0C0E14",
+    "tab.border": "#293045",
+    "editorGroup.background": "#0C0E14",
+    "editorGroupHeader.tabsBackground": "#0C0E14",
+    "editorGroup.border": "#293045",
+    "statusBar.foreground": "#9da5b4",
+    "statusBar.background": "#020304",
+    "statusBar.noFolderBackground": "#020304",
+    "statusBarItem.hoverBackground": "#2c313a",
+    "activityBar.background": "#0C0E14",
+    "activityBar.foreground": "#d7dae0",
+    "activityBarBadge.background": "#528bff",
+    "activityBarBadge.foreground": "#f8fafd",
+    "sideBar.background": "#0C0E14",
+    "sideBar.border": "#293045",
+    "sideBarSectionHeader.background": "#0C0E14",
+    "titleBar.activeForeground": "#9da5b4",
+    "titleBar.inactiveForeground": "#6b717d",
+    "titleBar.activeBackground": "#0C0E14",
+    "titleBar.inactiveBackground": "#0C0E14",
+    "notification.background": "#0C0E14",
+    "statusBar.debuggingBackground": "#020304",
+    "menu.selectionBackground": "#141721",
+    "menu.selectionForeground": "#d7dae0",
+    "checkbox.background": "#383e4a",
+    "panelSectionHeader.background": "#0C0E14",
+    "panelSectionHeader.border": "#fff",
```

Comment:
Should `"panelSectionHeader.border"` be set to this color? Seems like it should be `#293045` to match the rest of the theme.

Reply:
Ahh, Thanks! Nice catch :see_no_evil: I have changed the same on the parent branch. Release `v1.0.6` is live now :rocket: .

Reply:
Haha no problem, sorry for so many changes! Thanks for the theme! The updates you made look really good.

Reply:
I appreciate your changes and thank you so much for taking time for making the theme better :+1: